docgen/assets/img/InstantSearch-iOS.svg

```diff
@@ -0,0 +1,22 @@
+    </linearGradient>
+  </defs>
+  <g fill="none" fill-rule="evenodd" transform="translate(0 -2)">
+    <g fill-rule="nonzero" transform="translate(0 2)">
+      <path fill="url(#a)" d="M27.4367816,22.2888889 C28.7011494,22.2888889 29.9022989,22.037037 31.1034483,21.6592593 L33.1264368,25.1222222 L34.5172414,27.5148148 C34.8965517,28.2074074 35.591954,28.5851852 36.3505747,28.5851852 C36.7298851,28.5851852 37.045977,28.5222222 37.4252874,28.3333333 C38.4367816,27.7666667 38.7528736,26.4444444 38.183908,25.437037 L36.7931034,23.0444444 L34.7701149,19.5185185 C37.1091954,17.5037037 38.6264368,14.4814815 38.6264368,11.1444444 C38.6264368,5.03703704 33.6321839,0.062962963 27.5,0.062962963 C21.3678161,0.062962963 16.3103448,5.03703704 16.3103448,11.2074074 C16.3103448,17.3777778 21.3045977,22.2888889 27.4367816,22.2888889 Z M27.4367816,4.28148148 C31.2298851,4.28148148 34.3275862,7.36666667 34.3275862,11.1444444 C34.3275862,14.9222222 31.2298851,18.0074074 27.4367816,18.0074074 C23.6436782,18.0074074 20.545977,14.9222222 20.545977,11.1444444 C20.545977,7.36666667 23.6436782,4.28148148 27.4367816,4.28148148 Z"/>
+      <path fill="url(#b)" d="M43.8103448,28.2074074 C38.3735632,28.2074074 33.8850575,32.1111111 32.8735632,37.2111111 L28.8908046,37.2111111 L26.1091954,37.2111111 C24.9712644,37.2111111 24.0229885,38.1555556 24.0229885,39.2888889 C24.0229885,40.4222222 24.9712644,41.3666667 26.1091954,41.3666667 L28.8908046,41.3666667 L32.8735632,41.3666667 C33.8850575,46.4666667 38.3735632,50.3703704 43.8103448,50.3703704 C49.9425287,50.3703704 54.9367816,45.3962963 54.9367816,39.2888889 C54.9367816,33.1814815 49.9425287,28.2074074 43.8103448,28.2074074 Z M43.8103448,46.2148148 C40.0172414,46.2148148 36.9195402,43.1296296 36.9195402,39.3518519 C36.9195402,35.5740741 40.0172414,32.4888889 43.8103448,32.4888889 C47.6034483,32.4888889 50.7011494,35.5740741 50.7011494,39.3518519 C50.7011494,43.1296296 47.6034483,46.2148148 43.8103448,46.2148148 Z M18.4597701,30.9777778 C20.7988506,32.9925926 22.316092,35.9518519 22.2528736,39.3518519 C22.2528736,45.4592593 17.2586207,50.4333333 11.1264368,50.4333333 C4.99425287,50.4333333 0,45.5222222 0,39.3518519 C0,33.1814815 4.99425287,28.2074074 11.1264368,28.2074074 C12.454023,28.2074074 13.6551724,28.4592593 14.7931034,28.837037 L16.816092,25.3740741 L18.2068966,22.9814815 C18.8390805,21.9740741 20.1034483,21.6592593 21.1149425,22.2259259 C22.1264368,22.8555556 22.4425287,24.1148148 21.8735632,25.1222222 L20.4827586,27.5148148 L18.4597701,30.9777778 Z M11.1264368,46.2148148 C14.9827586,46.2148148 18.0804598,43.1296296 18.0172414,39.3518519 C18.0172414,35.5740741 14.9195402,32.4888889 11.1264368,32.4888889 C7.33333333,32.4888889 4.23563218,35.5740741 4.23563218,39.3518519 C4.23563218,43.1296296 7.33333333,46.2148148 11.1264368,46.2148148 Z"/>
+    </g>
+    <text fill="#3E396B" font-family="Montserrat-SemiBold, Montserrat" font-size="24" font-weight="500">
+      <tspan x="78" y="23">InstantSearch</tspan>
```

Comment:
I'm pretty sure that this will show as Times New Roman if you don't have Montserrat installed